nextstrain_profiles/nextstrain-gisaid/builds.yaml

```diff
@@ -4,6 +4,7 @@ auspice_json_prefix: ncov_gisaid
 custom_rules:
   - workflow/snakemake_rules/export_for_nextstrain.smk
   - nextstrain_profiles/nextstrain-gisaid/subsampling_ranges.smk
+  - workflow/snakemake_rules/nextstrain_auspice_config.smk
```

Comment:
As the new rule is part of export steps, it seems the rule could just be part of the existing `export_for_nextstrain.smk` file?

Reply:
Sure! I tend to think of that file more as an "upload assets" file, but it's actually more general than that so I'm happy to put the config JSON rule there.